site/gdocs/components/AtomArticleBlocks.tsx

```diff
@@ -50,13 +50,17 @@ function AtomArticleBlock({
         )
     }
     return match(block)
-        .with({ type: "image" }, (block) => (
-            <Image
-                filename={block.filename}
-                smallFilename={block.smallFilename}
-                alt={block.alt}
-            />
-        ))
+        .with({ type: "image" }, (block) => {
+            // Skip mobile-only blocks in Atom feeds
+            if (block.mobileOnly) return null
```

Comment:
This assumes that a mobileOnly image will only ever be used in concert with a desktopOnly image.